playground/src/pages/CodingEnvironment.tsx

```diff
@@ -260,9 +262,13 @@ export default function CodingEnvironment(): JSX.Element {
       {showDebug ? (
         <div className="flex justify-center pt-6">
```

Comment:
The buttons borders are not rounded to the right direction:

![image](https://user-images.githubusercontent.com/310139/227037953-e22b5c52-8040-4691-a8cd-06407760b760.png)

the left one should be `rounded-l-md`, the middle ones `rounded-none`, the right one `rounded-r-md`

Reply:
btw, which version of the buttons is being used? this one, or the one in `playground/src/components/DebugButtons.tsx`?

Reply:
Good point I moved the buttons down and squared them